src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.card__star-container {
+  display: flex;
+}
+
+.card__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  @include starSize($star-size);
+}
+
+.card__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__star-container--star_1 > :nth-child(-n + 1),
+.card__star-container--star_2 > :nth-child(-n + 2),
+.card__star-container--star_3 > :nth-child(-n + 3),
+.card__star-container--star_4 > :nth-child(-n + 4),
+.card__star-container--star_5 > :nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
We should use loop here. I understand this after today lesson))

Reply:
I also had the same thoughts :)